megamek/src/megamek/common/Aero.java

```diff
@@ -1609,6 +1609,17 @@ public int getThresh(int loc) {
         return 0;
     }
 
+    @Override
+    public int getHighestThresh() {
+        int max = damThresh[0];
+        for (int i = 1; i < damThresh.length; i++) {
+            if (damThresh[i] > max) {
+                max = damThresh[i];
+            }
+        }
+        return max;
```

Comment:
Would like to have a sanity check to make sure this is the correct way to find this value for the errata